test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/DistributedCacheTagHelperTest.cs

```diff
@@ -753,6 +753,7 @@ public void UpdateCacheEntryOptions_SetsSlidingExpiration_IfExpiresSlidingIsSet(
             var childContent = "some-content";
             var resetEvent1 = new ManualResetEvent(false);
             var resetEvent2 = new ManualResetEvent(false);
+            var resetEvent3 = new ManualResetEvent(false);
```

Comment:
This covers the last test using `ManualResetEvent`s in this class. Does `CacheTagHelperTest` need a similar fix?